app/policies/follow_request_policy.rb

```diff
@@ -0,0 +1,19 @@
+class FollowRequestPolicy < ApplicationPolicy
+  attr_reader :user, :follow_request
+
+  def index?
+    true
+  end
+
+  def create?
+    true
+  end
+
+  def update? 
+    true
```

Comment:
there should probably be some logic here, to prevent third users to temper with a follow request. For example `user == follow_request.recipient || user == follow_request.sender`